lib/python/qmk/cli/console.py

```diff
@@ -0,0 +1,285 @@
+"""Acquire debugging information from usb hid devices
+
+cli implementation of https://www.pjrc.com/teensy/hid_listen.html
+"""
+from pathlib import Path
+from threading import Thread
+from time import sleep, strftime
+
+import hid
+import usb.core
+
+from milc import cli
+
+LOG_COLOR = {
+    'next': 0,
+    'colors': [
+        '{fg_blue}',
+        '{fg_cyan}',
+        '{fg_green}',
+        '{fg_magenta}',
+        '{fg_red}',
+        '{fg_yellow}'
+    ]
+}
+
+KNOWN_BOOTLOADERS = {
+        # VID  ,  PID
+        ('16C0', '0478'): 'Teensy Halfkay Bootloader',
+        ('0483', 'DF11'): 'STM32 DFU',
+        ('314B', '0106'): 'APM32 DFU',
+        ('1EAF', '0003'): 'STM32duino Bootloader',
+        ('16C0', '05DC'): 'USBaspLoader',
+        ('1C11', 'B007'): 'Kiibohd DFU',
+        ('03EB', '2FEF'): 'ATmega16U2 DFU',
+        ('03EB', '2FF0'): 'ATmega32U2 DFU',
+        ('03EB', '2FF3'): 'ATmega16U4 DFU',
+        ('03EB', '2FF4'): 'ATmega32U4 DFU',
+        ('03EB', '2FF9'): 'AT90USB64 DFU',
+        ('03EB', '2FFA'): 'AT90USB162 DFU',
+        ('03EB', '2FFB'): 'AT90USB128 DFU'
```

Comment:
Missing Caterina and SAM-BA here.

Reply:
https://docs.qmk.fm/#/driver_installation_zadig?id=list-of-known-bootloaders

Reply:
SAM-BA is `03EB:6124`

Reply:
I've added these in ab0d7a378.